controllers/user.js

```diff
@@ -28,9 +38,25 @@ const register = async (req, res, next) => {
       r: "pg",
       d: "identicon",
     });
-    const newUser = new User({ username, email, avatarURL });
+    const verificationToken = nanoid();
+    const newUser = new User({ email, avatarURL });
+    newUser.verificationToken = verificationToken;
+    newUser.verify = false;
```

Comment:
To powinno być ustawione default na false zeby nie trzeba bylo robic tego tutaj.